src/markdown.c

```diff
@@ -1403,7 +1403,7 @@ prefix_quote(uint8_t *data, size_t size)
 	if (i < size && data[i] == ' ') i++;
 	if (i < size && data[i] == ' ') i++;
 
-	if (i < size && data[i] == '>') {
+	if ((i < size && data[i] == '>') && (i + 1 < size && data[i+1] != '!')) {
```

Comment:
Just out of curiosity, did you consider trying to shoehorn the spoiler stuff straight into the quote work that already existed rather than creating new methods for it? Seems like it might fit, even though I am totally unsure whether that would actually be a good idea or not.